components/tools/OmeroWeb/omeroweb/decorators.py

```diff
@@ -457,6 +457,9 @@ def wrapped(request, *args, **kwargs):
                     ctx.load_server_settings(conn, request)
 
                     share_id = kwargs.get('share_id')
+                    if share_id is None:
```

Comment:
I know what share_id is for. bu allowing ```share_id``` in query string you allow to establish share connection for every URL in webclient. That is a bug.

As discussed:
```
GET https://server/webclient/api/images/?share_id=...
```

should be replaced by 

```
https://server/webclient/api/share/share_id/
```